code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -146,7 +146,7 @@
 	name = "Cryoxadone"
 	description = "A chemical mixture with almost magical healing powers. Its main limitation is that the patient's body temperature must be under 270K for it to metabolise correctly."
 	color = "#0000C8"
-	taste_description = "sludge"
+	taste_description = "blue gatorade"
```

Comment:
this is a licensed name, can't use it

Reply:
ok what name would be suitable then

Reply:
spacerade

Reply:
blue salty water

Reply:
blue energy drink?

Reply:
blue [energy drink you aren't allowed to think about without actually drinking it due to intellectual property laws]

Reply:
What about just `blue`?

Reply:
blue it is then